bundles/io/org.eclipse.smarthome.io.console.rfc147/src/main/java/org/eclipse/smarthome/io/console/rfc147/internal/CommandHelp.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.io.console.rfc147.internal;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.smarthome.io.console.Console;
+import org.eclipse.smarthome.io.console.ConsoleInterpreter;
+import org.eclipse.smarthome.io.console.extensions.GenericConsoleCommandExtension;
+
+/**
+ *
+ * @author Markus Rathgeb - Initial contribution and API
+ *
+ */
+public class CommandHelp extends GenericConsoleCommandExtension {
```

Comment:
Why not also name it "HelpConsoleCommandExtension" to be aligned with the other extensions?